atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/backup/CassandraRepairHelper.java

```diff
@@ -121,7 +121,9 @@ public void repairTransactionsTables(
                 getRangesForRepairByTable(atlasService, transactionsTableInteractions);
 
         tokenRangesForRepair.forEach((table, ranges) -> {
-            log.info("Repairing ranges for table", SafeArg.of("table", table));
+            log.info("Repairing ranges for table",
+                    SafeArg.of("namespace", atlasService.getNamespace()),
```

Comment:
[spotless wants a linebreak](https://app.circleci.com/pipelines/github/palantir/atlasdb/8594/workflows/c52f22f0-339a-47eb-b035-b8d491f74b04/jobs/33896)

```suggestion
            log.info(
                    "Repairing ranges for table",
                    SafeArg.of("namespace", atlasService.getNamespace()),
```


```
org.gradle.api.GradleException: The following files had format violations:
    src/main/java/com/palantir/atlasdb/cassandra/backup/CassandraRepairHelper.java
        @@ -121,7 +121,8 @@
         ················getRangesForRepairByTable(atlasService,·transactionsTableInteractions);
         
         ········tokenRangesForRepair.forEach((table,·ranges)·->·{
        -············log.info("Repairing·ranges·for·table",
        +············log.info(
        +····················"Repairing·ranges·for·table",
         ····················SafeArg.of("namespace",·atlasService.getNamespace()),
         ····················SafeArg.of("table",·table));
         ············repairTable.accept(table,·ranges);
Run './gradlew :atlasdb-cassandra:spotlessApply' to fix these violations.
```